Tests/HaCWebsiteLibTests/HaCWebsiteLibTestCase.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import XCTest
+
+class HaCWebsiteLibTestCase: XCTestCase {
```

Comment:
I like that we're creating reusable test code, but is there any reason why we're using inheritance here? (e.g. is there protected state that we need to access for these to work) In general, inheritance adds incidental complexity and from what I can tell, is not very Swift-ey outside of the context of iOS.

I know we need to inherit from `XCTestCase` to make tests work, but we don't need to add any more complexity if we don't need to.